systems/subscriber/api-gateway/pkg/rest/api.go

```diff
@@ -4,15 +4,17 @@ import (
 	"time"
 
 	"github.com/google/uuid"
+	"google.golang.org/protobuf/types/known/timestamppb"
 )
 
 type SimInfo struct {
-	Iccid          string `json:"iccid" validate:"required"`
-	SimType        string `json:"simType" validate:"required"`
-	Msidn          string `json:"msidn" validate:"required"`
-	SmDpAddress    string `json:"smdpAddress" validate:"required"`
-	ActivationCode string `json:"activationCode" validate:"required"`
-	QrCode         string `json:"qrcode" validate:"required"`
+	Iccid          string `json:"iccid" binding:"required" validate:"required"`
+	SimType        string `json:"simType" binding:"required" validate:"required"`
```

Comment:
I think the 2 words json aliases are snake_cased instead of being camelCased eg. `json:"sim_type"`  instead of `json:"simType"`

Reply:
Addressed